client/components/jetpack/backup-card/style.scss

```diff
@@ -39,6 +39,42 @@ $cloud-icon-height: 24px;
 	.is-featured & {
 		margin-bottom: 100px;
 	}
+
+	&.backup-card__title--scheduled {
+		color: var( --studio-gray-40 );
+		margin-top: 5px;
```

Comment:
It seems like these `margin-top` are unnecessary. Let me know if that's not the case.

Reply:
Yes you're right. Nice catch. There's 4 like this, I've removed them. 👍